draft-ietf-tls-tls13.md

```diff
@@ -4949,6 +4980,12 @@ server is not able to guarantee full uniqueness of the handshake
 state. See {{early-data-indication}} for one mechanism to limit
 the exposure to replay.
 
+KCI resistance describes the security guarantees that a party may still have
+after its long-term secret has been compromised. TLS does not provide security
+for data communicated after the peer's long-term secret (signature key or
+external PSK) is compromised. It therefore does not provide post-compromise
+security {{CCG16}}, sometimes also referred to as backwards or future security.
```

Comment:
1. You start out by describing KCI but then don't say what TLS's behavior with respect to KCI is, but rather talk about security with respect to the *peer*'s long term secret rather than the local one.

2. This text doesn't seem quite correct. Consider the following sequence of events:

- TLS handshake between A and B using ECDHE
- A's long-term secret is compromised
- A sends message "X" to B

The compromise of A's long-term secret does not (as far as I know) threaten either the confidentiality or integrity of "X".

I think you need to split this up into compromises which happen before the handshake and those which happen after the handshake but before data is sent on the channel